CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- It is recommended that prior to developing enhancements to this charm
+  an issue explaining the use case is created at the official
+  Canonical [github repository](https://github.com/canonical/prometheus-operator).
+- Should you require any assistance in exploring use cases or
+  discussing proposed implementation you may directly engage with
+  other developers in the Canonical
+  [public channel](https://chat.charmhub.io/charmhub/channels/charm-dev) using
+  [Mattermost](https://mattermost.com/).
+- It is strongly recommended that prior to engaging in any enhancements
+  to this charm you familiarise your self with the Juju
+  [SDK](https://juju.is/docs/sdk) and
+  [Operator Framework](https://ops.readthedocs.io/en/latest/) documentation.
+- All enhancements require a code review after creating a pull request
```

Comment:
- All enhancements require review before being merged. Besides the code quality and test coverage, the review will also take into account the resulting user experience for Juju administrators using this charm. Please help us out in having easier reviews by rebasing onto the `main` branch, avoid merge commits and enjoy a linear Git history.

Reply:
Fixed.